util/src/main/i18n/templates/map.properties

```diff
@@ -119,3 +119,29 @@ vote.against = Voted against {0}
 
 # {0} = option name (e.g. "Airship Battle")
 vote.abstain = Removed your vote for {0}
+
+vote.disabled = Voting is disabled
+
+# {0} = player name (who added the map)
+# {1} = map name 
+vote.custom.add = {0} added {1} to the custom vote
+
+# {0} = player name (who removed the map)
+# {1} = map name 
+vote.custom.remove = {0} removed {1} from the custom vote
+
+# {0} = map name
+vote.custom.existing = {0} has already been added to the custom vote
+
+# {0} = player name (who cleared the map selection)
+vote.custom.clear = {0} has cleared the custom vote selection
+
+# {0} = map name
+vote.custom.notFound = {0} could not be removed since it was never added
+
+vote.custom.inProgress = Voting selection can not be modified at this time
+
+vote.custom.limit = Map limit for the custom vote has been reached
+
+vote.custom.list = Selected Maps
```

Comment:
`vote.title.map = Selected Maps`

In the future we could have `vote.title.gamemode`